Generals/CMakeLists.txt

```diff
@@ -3,8 +3,10 @@ cmake_minimum_required(VERSION 3.25)
 project(g_generals LANGUAGES C CXX)
 
 # Do we want to build extra SDK stuff or just the game binary?
-option(GENZH_BUILD_GENERALS_TOOLS "Build tools for Generals" OFF)
+option(GENZH_BUILD_GENERALS_TOOLS "Build tools for Generals" ON)
 add_feature_info(GeneralsTools GENZH_BUILD_GENERALS_TOOLS "Build Generals Mod Tools")
+option(GENZH_BUILD_GENERALS_EXTRAS "Build extra stuff for Generals" OFF)
```

Comment:
Perhaps clarify that "stuff" also refers to tools, but less useful ones.

"Build additional less useful tools for Generals"